src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewFilterMapper.xml

```diff
@@ -139,16 +139,16 @@
             <!-- TODO: add patient level data filtering -->
             <if test="studyViewFilterHelper.studyViewFilter.genomicDataFilters != null and !studyViewFilterHelper.studyViewFilter.genomicDataFilters.isEmpty()">
                 <foreach item="genomicDataFilter" collection="studyViewFilterHelper.studyViewFilter.genomicDataFilters" open="INTERSECT" separator="INTERSECT">
-                    <choose>
-                        <!-- Currently we only handle CNA in categorical genomic data filters. Future categorical profile types will involve updates to this part-->
-                        <when test="genomicDataFilter.profileType == 'cna' or genomicDataFilter.profileType == 'gistic'">
-                            <include refid="categoricalGenomicDataFilterForCNA"/>
-                        </when>
-                        <otherwise>
-                            <!-- when you make selection in genomic binning chart, it will still send genomic data filter -->
-                            <include refid="numericalGenomicDataFilter"/>
-                        </otherwise>
+                    (
```

Comment:
@fuzhaoyuan i think the problem was that we introduced a UNION ALL inside in order to handle the nulls and that needs to be scoped so as to avoid interfering with the INTERSECTION above it.

Reply:
probably needs to be applied also to generic assay